Android/plugin-manifest.json

```diff
@@ -56,5 +56,21 @@
             "tooltip_text": "The text on the button that appears alongside a connectivity error message",
             "default": "Try Again"
         }
-    ]
+    ],
```

Comment:
What about the "ignore_default_subscription" part?

Reply:
Added.